docs/src/2.3.RenderingObjects.mdx

```diff
@@ -0,0 +1,199 @@
+import CodeSandboxEmbed from './jsx/utils/CodeSandboxEmbed.js';
+
+# Rendering Objects
+
+In this step, you'll learn how to:
+
+- Build a Hello World app with `regl-worldview`
+- Render objects with custom position, color, and scale
+- Apply math equation to the rendered objects
+- Use instanced rendering
+
+## Building a Hello World App
+
+Add `regl-worldview` to your package dependencies:
+
+```json
+"regl-worldview": "latest"
+```
+
+Create a new file titled `Example.js` in the `src/` directory, and add the code below which renders `Worldview` with a red cube and x/y/z axes:
+
+```js
+// src/Example.js
+import React from "react";
+import Worldview, { Cubes, Axes } from "regl-worldview";
+
+export default function Example() {
+  return (
+    <Worldview>
+      <Cubes>
+        {[
+          {
+            pose: {
+              orientation: { x: 0, y: 0, z: 0, w: 1 },
+              // position the cube at the center
+              position: { x: 0, y: 0, z: 0 },
+            },
+            scale: { x: 10, y: 10, z: 10 },
+            // rgba values are between 0 and 1 (inclusive)
+            // http://wiki.ros.org/rviz/DisplayTypes/Marker
+            color: { r: 1, g: 0, b: 0, a: 1 },
+          },
+        ]}
+      </Cubes>
+      <Axes />
+    </Worldview>
+  );
+}
+```
+
+Replace the contents of `src/index.js` with the following code to render your `Example` component:
+
+```js
+// src/index.js
+import React from "react";
+import ReactDOM from "react-dom";
+import Example from "./Example";
+
+function App() {
+  return (
+    <div style={{ width: "100vw", height: "100vh" }}>
```

Comment:
In my browser the viewport goes up and down when scrolling up and down, because `<body>` has a margin by default. Might be nice to remove that margin using `document.body.style.margin = 0;`

Reply:
Actually that doesn't fully solve it for me. Maybe it's easier to just use `90vw` and `90vh` or so?

Reply:
Seems related to latest Chrome update. Will update in another PR